### ISSUE TITLE
maven new project wizard should define minimum maven version

### ISSUE THREAD

**Issue by elharo** [GoogleCloudPlatform/google-cloud-eclipse#3393]
App Engine docs specify 3.5.0 so that's probably what we should set. 

```
[WARNING] Project does not define minimum Maven version, default is: 2.0
[INFO] Plugins require minimum Maven version of: 3.0
[INFO] Note: the super-pom from Maven 3.5.0 defines some of the plugin
[INFO]       versions and may be influencing the plugins required minimum Maven
[INFO]       version.
[INFO] 
[ERROR] Project does not define required minimum version of Maven.
[ERROR] Update the pom.xml to contain
[ERROR]     <prerequisites>
[ERROR]       <maven>3.0</maven>
[ERROR]     </prerequisites>
[INFO] 
```

**Comment by briandealwis** [comment]
So we had that in CT4E projects and dropped it as Maven complains loudly that `<prerequisites>` is only intended for `maven-plugin` projects.  Full details in GoogleCloudPlatform/google-cloud-eclipse#2966.

**Comment by elharo** [comment]
Hmm, so why am I seeing that warning now?

**Comment by briandealwis** [comment]
I've always seen that message from the versions-maven-plugin.

Aha, it looks like this was fixed in `versions-maven-plugin` 2.6 (https://github.com/mojohaus/versions-maven-plugin/pull/230).  It now knows how to retrieve the maven versions from the enforcer plugin.

Apparently this is what's required.  A bit verbose!

https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
